app/styles/ui/_diff.scss

```diff
@@ -286,17 +286,36 @@
   }
 }
 
-.image-diff-two-up {
+.image-diff-container {
   display: flex;
   flex: 1;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
 
-  .image-diff-previous {
-    margin-right: 7.5px;
+  .image-diff-two-up {
+    display: flex;
+    align-items: center;
+
+    .image-diff-previous {
+      margin-right: 7.5px;
+      margin-bottom: 7.5px;
```

Comment:
Is there a reason why you're using a half pixel? I tend to avoid them since they're specific to high density monitors and haven't found a compelling reason to use them instead of a whole number.

Also, are you trying to apply this margin to create space between the diff size summary below the two images? If so, it might make sense move this margin under `.image-diff-two-up` on Line 296  and remove this line and 306. This should have the same visual effect while adding the benefit of having to change code in one place.

Reply:
`7.5` pixel was the original code; I just added this fragment to the container (that's why it's modifed). I wanted to position the `Diff: xx%` below properly. The idea behind the margin-right/bottom is to position the images in somehow in the "center" of the views. Please have a look at the image below:

![diff](https://user-images.githubusercontent.com/988790/39868868-bdce1678-545b-11e8-890e-b9f896e21e93.png)